packages/decentraland-loader/lifecycle/controllers/download.ts

```diff
@@ -71,13 +75,60 @@ export class SceneDataDownloadManager {
     }
   }
 
+  createFakeILand(sceneId: string) {
+    const coordinates = sceneId.replace('empty-', '')
+    const pick = ((((coordinates.split(',').reduce((prev, next) => prev + parseInt(next, 10), 0) % 12) + 13) % 12) + 1)
```

Comment:
Do we actually need the `($ + 13) % 12` part?
```suggestion
    const pick = ((coordinates.split(',').reduce((prev, next) => Math.abs(prev + parseInt(next, 10), 0) % 12)) + 1)
```